src/main/webapp/js/user-page-loader.js

```diff
@@ -49,7 +49,12 @@ function showMessageFormIfLoggedIn() {
 
 /** Fetches messages and add them to the page. */
 function fetchMessages() {
+  //const url = '/messages?user=' + parameterUsername;
+  const parameterLanguage = urlParams.get('language');
```

Comment:
Change const to let, this will remove the exception on line 56 where the code is trying to modify a constant